### DISCUSSION TITLE
Real time dashboard - multipe NICs

### DISCUSSION THREAD

**Discussion by pixel-paul** [webmin/authentic-theme#1680] [General]
Is it possible to display multiple NIC throughput on the real time dashboard?

**Comment by iliaross** [comment]
Do you mean in graphs?

**Comment by pixel-paul** [comment]
Yes, so the 'Stats History':

![image](https://github.com/webmin/authentic-theme/assets/389303/4c331aa6-1f4a-4c27-aa60-e9f7d61d467e)

**Comment by iliaross** [comment]
As we show Network I/O already, the question is, does it represent combined data from both NICs?

**Comment by pixel-paul** [comment]
Aha, it is I/O isn't it - and I have no idea!
